h/static/scripts/directive/test/annotation-test.js

```diff
@@ -1404,6 +1175,26 @@ describe('annotation', function() {
           assert.ok(controller.editing());
         });
       });
+
+      it(
```

Comment:
As mentioned before, I think worrying about long lines in tests is a recipe for a tedious life.

```javascript
it('Passes group:<id> to the server when saving a new annotation', function () {
  ...
})
```